Src/Support/Google.Apis.Auth/OAuth2/LocalServerCodeReceiver.cs

```diff
@@ -548,15 +579,29 @@ internal CallbackUriChooser(IClock clock, TimeSpan timeout, Func<string, bool> l
                 _listenerFailsFor = listenerFailsFor;
             }
 
-            public string GetUriTemplate()
+            public string GetUriTemplate(CallbackUriChooserStrategy strategy)
```

Comment:
Maybe make this method internal, to make it clearer in code review that it's okay to make breaking signature changes? (I had to expand the diff to include the class definition.)